common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -977,6 +977,13 @@ def test_course_structure_cache(self, mock_get_cache):
         # now make sure that you get the same structure
         self.assertEqual(cached_structure, not_cached_structure)
 
+        # If data is corrupted, get it from mongo again.
+        cache_key = self.new_course.id.version_guid
+        self.cache.set(cache_key, b"bad_data")
+        with check_mongo_calls(1):
+            not_cached_structure = self._get_structure(self.new_course)
```

Comment:
Don't we want to check something about `not_cached_structure`?  It should be real data, right?

Reply:
Added an equality check as well.